### DISCUSSION TITLE
support VMess AEAD / alterid

### DISCUSSION THREAD

**Discussion by sbh96** [eycorsican/leaf#321] [Ideas]
hi please support vmess aead because vmess protocol without alterid is nothing,
if you don't update you lost the biggest v2ray users

**Comment by yeshaohuapro** [comment]
"VMessAEAD is enforced and a non VMessAEAD connection is received."
hi please support vmess aead.

**Reply by ghost** [reply]
https://github.com/Web3123/v2ray 这是项目地址
